tests/english_member_page_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/english_members_page.rb'
+
+describe EnglishMembersPage, :vcr do
+  describe 'member_urls' do
+    it 'should return the expected number of urls' do
+      url = 'http://www.althingi.is/altext/cv/en/'
+      EnglishMembersPage.new(response: Scraped::Request.new(url: url).response)
```

Comment:
same here — if every test is going to use the same Page, better to move it out, and keep the body of the test for things that will vary.